gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -233,11 +236,15 @@ func (p *glusterBlockProvisioner) Provision(options controller.VolumeOptions) (*
 
 	if cfg.chapAuthEnabled && iscsiVol.User != "" && iscsiVol.AuthKey != "" {
 		nameSpace := options.PVC.Namespace
```

Comment:
should we default to a dedicated namespace and ditch the PVC namespace idea altogether?

Reply:
hmmm.. then this parameter will become `mandatory` than `optional`. I am not sure whether pod spec has any restriction to use a PV which has reference to a secret from a different namespace. If not, we can mandate this and forget about `pvc namespace` altogther. @rootfs any thoughts ?

Reply:
not necessarily mandatory, could default to `kube-system` or something? but yeah, need second opinions.